<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormNhandien</class>
 <widget class="QWidget" name="FormNhandien">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#bt_back{
background-color:rgba(2,65,118,255);
color:rgba(255,255,255,200);
border-radius:5px;
}
QPushButton#bt_back:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2,65,118,100);
background-position:calc(100%-10px)center;
}
QPushButton#bt_back:hover{
background-color:rgba(2,65,118,200);
}
QPushButton#bt_camera{
background-color:rgba(2,65,118,255);
color:rgba(255,255,255,200);
border-radius:5px;
}
QPushButton#bt_camera:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2,65,118,100);
background-position:calc(100%-10px)center;
}
QPushButton#bt_camera:hover{
background-color:rgba(2,65,118,200);
}
QPushButton#bt_start{
background-color:rgba(2,65,118,255);
color:rgba(255,255,255,200);
border-radius:5px;
}
QPushButton#bt_start:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2,65,118,100);
background-position:calc(100%-10px)center;
}
QPushButton#bt_start:hover{
background-color:rgba(2,65,118,200);
}
QPushButton#bt_end{
background-color:rgba(2,65,118,255);
color:rgba(255,255,255,200);
border-radius:5px;
}
QPushButton#bt_end:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2,65,118,100);
background-position:calc(100%-10px)center;
}
QPushButton#bt_end:hover{
background-color:rgba(2,65,118,200);
}
QPushButton#bt_xemlich{
background-color:rgba(2,65,118,255);
color:rgba(255,255,255,200);
border-radius:5px;
}
QPushButton#bt_xemlich:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2,65,118,100);
background-position:calc(100%-10px)center;
}
QPushButton#bt_xemlich:hover{
background-color:rgba(2,65,118,200);
}
QPushButton#bt_trogiup{
background-color:rgba(2,65,118,255);
color:rgba(255,255,255,200);
border-radius:5px;
}
QPushButton#bt_trogiup:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2,65,118,100);
background-position:calc(100%-10px)center;
}
QPushButton#bt_trogiup:hover{
background-color:rgba(2,65,118,200);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/backgroud/backgroud/1614776.jpg);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Camera_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>531</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/backgroud/backgroud/9.jpg);
border- radius:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_camera">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>UVN Dzung Dakao</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Bật camera</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>331</width>
     <height>69</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="TimeLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Time:</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Time_Lable">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="DateLable">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Date_Lable">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>331</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color:rgba(0,0,0,75);</string>
   </property>
   <property name="title">
    <string>Thông tin:</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>311</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 0);</string>
         </property>
         <property name="text">
          <string>Tên:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Ten_Label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Id_Label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Trạng thái:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Trangthai_Label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="bt_back">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>550</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>UVN Dzung Dakao</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_start">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>UVN Dzung Dakao</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Bắt đầu </string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_end">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>UVN Dzung Dakao</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Kết thúc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_xemlich">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>480</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>UVN Dzung Dakao</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Xem lịch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_trogiup">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>480</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>UVN Dzung Dakao</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Trợ giúp</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
